proposals/src/passive-stake-delegation-and-rewards.md

```diff
@@ -58,10 +58,27 @@ staker's StakeState accounts.  This is the percentage ceiling of the reward.
 * Account::lamports - The accumulated lamports from the commission.  These do not
 count as stakes.
 
-* `authorized_voter_id` - Only this identity is authorized to submit votes.
+* `authorized_vote_signer` - Only this identity is authorized to submit votes, and
```

Comment:
Can we delete this field and have the vote signer create this account?

Reply:
if we do so, we'll need a field for the validator node's pubkey (who can submit votes?).  using this approach allows us to bypass needing a vote_signer by default

Reply:
I'm confused. Why does the validator node submit votes instead of the vote signer?  I think if there's any default, it should be to use the Vote account's pubkey as the validator's pubkey.

Reply:
the validator node ships the vote over gossip.  the (optional) vote signer merely presents KeypairUtil to the node

Reply:
https://github.com/rob-solana/solana/blob/24dca44e48134a1cc19ff306107224a51e0e228d/consensus.svg

Reply:
here's the hack for the race: https://github.com/solana-labs/solana/blob/8d032aba9d978df32215cf35867c9afac45c3bca/core/src/staking_utils.rs#L71

Reply:
@rob-solana, It's only the use of `VotingKeypair` (`RemoteVoteSigner`) that's optional.  There will always be a Vote account. The keypair for that Vote account is what you'd put into `RemoteVoteSigner`.  On validator startup,  it registers its own pubkey with RemoteVoteSigner, but that's just to get it over to RemoteVoteSigner. The RemoteVoteSigner then re-signed the transaction with its keypair, which is the owner of the Vote account.

tldr; a big appeal of this design proposal is that `authorized_vote_signer` is no longer needed.

Reply:
Regarding your hack, why not add `config-staking-account` options to `create-staking-account` in solana-wallet (and then send a single transaction)?

Reply:
we tried that earlier...  the issue was when we needed which keypairs

the decision to pursue that hack for Beacons was expediency, and knowing that this proposal was out there, waiting